samples/fpgaperf_counter/fpgaperf_counter.h

```diff
@@ -47,6 +47,7 @@ extern "C" {
 
 #define FPGA_PERF_MAGIC 	0x46504741584c474c
 #define DFL_PERF_STR_MAX	256
+#define CAP_FPGA_PERFMON	38
```

Comment:
if this could be defined in other versions of the libcap header, then maybe we should check if it's defined before redefining it here.